### ISSUE TITLE
Double announcement of Qt dialogs

### ISSUE THREAD

**Issue by CoBC** [nvaccess/nvda#14560]
I report this here because I can't reproduce the issue with Narrator, but if you think it's a Qt issue, I'll report to Qt.

### Steps to reproduce:
* Open any Qt based application (VLC, OBS, TeamTalk)
* If necessary, open a dialog of this app (but not always necessary to reproduce).

### Actual behavior:

- NVDA reads dialog and focused item twice, for example here with TT:
"TeamTalk v. 5.13.0.5108 - Unreleased window. TeamTalk v. 5.13.0.5108 - Unreleased window."

### Expected behavior:

- NVDA should also announce one time dialog and item, for example: "TeamTalk v. 5.13.0.5108 - Unreleased window."

#### NVDA installed/portable/running from source:
- Installed

#### NVDA version:
- alpha-27541,e4608014 (2023.1.0.27541)

#### Windows version:
- Windows 11 22H2 (AMD64) build 22621.1105

#### Name and version of other software in use when reproducing the issue:
- Any Qt based application (tested with TT 5.13 beta and OBS 29.0)

#### Does the issue still occur after restarting your computer?
- Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
- Same with 2022.4

#### If NVDA add-ons are disabled, is your problem still occurring?
- Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
- Not tested but probably not related

**Comment by amirmahdifard** [comment]
@seanbudd @michaelDCurran @josephsl do you really think that this problem is related to nvda or Qt? i think it is not, because mostely i see this kind of problems in Qt teamtalk, and not with other Qt applications, but anytime i report this kind of problems in teamtalk, the developers are saying that it is Qt problem, but what do you think? i think that may be it is just depends on how you code the application, and not a qt problem, i mean, it could be qt problem sometimes, but teamtalk developers are saying that mostely it is qt problem, but i can think that they can be not true, because, for example, in server list dialogue, the edit box for filling username and password and nickname are not accessable so that nvda cannot review the text in those edit box's, but the other edit box's are properly accessible, so i think in this kace, it is not a qt problem.

**Comment by CoBC** [comment]
@amirmahdifard Seriously, I know what I say, if I tell you that an issue is related to Qt, it's because I know Qt bugs and I reported the issue to Qt devs. For the edit boxes problem you are talking about here, I already told you that the issue has been fixed on the Qt side for Qt 6.5 beta and 6.6-dev, and for now TT use Qt 6.5-dev, the edit boxes on server list are not simple edit boxes but editable comboboxes, and yes it's not accessible like other edit boxes for now. I already fixed a lot of accessibility problems in TT but I can't do all things because yes, even if you don't want, some issues are related to Qt and/or NVDA. Anyway it's not the right place to talk about TT specific problems here so I don't understand why you mensioned all contributors of this repo. It will be really appreciated if you let devs work normally sometimes in the future.
